.drone.yml

```diff
@@ -0,0 +1,25 @@
+clone:
```

Comment:
I don't think you need a custom git clone here, drone's default one should be enough ;)